test/base.js

```diff
@@ -14,6 +14,8 @@ var Promise = require('pinkie-promise');
 var yeoman = require('yeoman-environment');
 var userHome = require('user-home');
 
+var LF = process.platform === 'win32' ? '\r\n' : '\n';
```

Comment:
@addyosmani hmm do you think it can help with CI environments ? Didn't thought about it.